web/src/features/AppConfig/components/AppConfig.tsx

```diff
@@ -76,18 +76,19 @@ type State = {
   activeGroups: string[];
   app: App | null;
   changed: boolean;
-  configError: boolean;
+  configErrorMessage: string;
   configGroups: ConfigGroup[];
   configLoading: boolean;
   displayErrorModal: boolean;
   downstreamVersion: Version | null;
   errorTitle: string;
   gettingConfigErrMsg: string;
-  showValidationError: boolean;
   initialConfigGroups: ConfigGroup[];
   savingConfig: boolean;
+  showConfigError: boolean;
```

Comment:
`configError` -> `showConfigError` - probably don't need both of these but it's more explicit to have both.

Reply:
it's good to be more explicit, easier to read imo